### PR TITLE
[IMP] planner: hide progress bar when done

### PR DESCRIPTION
**Pull request by vidhin6**
TASK ID: https://www.odoo.com/web#id=12103&view_type=form&model=project.task&action=333&active_id=131
PAD: https://pad.odoo.com/p/r.zpv9zvWe3By334jk

### CODE REVIEW

File: addons/planner/static/src/js/planner.js [hunk 1]
```diff
@@ -68,6 +68,11 @@
             this.$(".oe_planner_progress").tooltip({html: true, title: this.planner.tooltip_planner, placement: 'bottom', delay: {'show': 500}});
             this.dialog.on("planner_progress_changed", this, function(percent){
                 self.update_parent_progress_bar(percent);
+                if (percent == 100) {
+                    $('#progress').hide();
+                }else{
+                    $('#progress').show();
+                }
```
Comment (vme-odoo) [inline addons/planner/static/src/js/planner.js]: Why using global selector?

File: addons/planner/static/src/xml/planner.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
        <li class="oe_topbar_item oe_planner_systray">
             <div class="oe_planner_progress">
                 <span class="fa fa-question-circle pull-left" style="color:white;"/>
-                <div class="progress">
+                <div class="progress" id="progress">
```
Comment (vme-odoo) [inline addons/planner/static/src/xml/planner.xml]: No need to define id.

File: addons/planner/static/src/js/planner.js [hunk 3]
```diff
@@ -68,6 +68,12 @@
             this.$(".oe_planner_progress").tooltip({html: true, title: this.planner.tooltip_planner, placement: 'bottom', delay: {'show': 500}});
             this.dialog.on("planner_progress_changed", this, function(percent){
                 self.update_parent_progress_bar(percent);
+                var $bar = this.$(".progress")
+                if (percent == 100) {
+                    $bar.hide(300);
+                } else {
+                    $bar.show(300);
+                }
```
Comment (vme-odoo) [inline addons/planner/static/src/js/planner.js]: use display none and improve condtion

**Comment by dmo-odoo** [comment]
Merged a modified version which fixes the proressbar css nonsense to be able to do show/hide on the progressbar itself without applying element-level css style at https://github.com/odoo/odoo/commit/d1e4e0d58dba0130816cb078c94513e12c85d5ce.

**Pull request closed without merging**